xml/mysql_banners.xml

```diff
@@ -759,7 +760,7 @@
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
   <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~jessie(?:-log)?$" flags="REG_ICASE">
-    <description>MariaDB MariaDB on Debian 8.0 (jessie)</description>
+    <description>MariaDB on Debian 8.11 (jessie)</description>
```

Comment:
Why the change from `8.0` to `8.11`?

Reply:
that was a mistake i meant to remove all the point values before pushing